enocean/eep_parser.py

```diff
@@ -105,14 +105,17 @@ def _parse_eep_A5_04_02(self, payload, status):
         result = {}
         result['ENG'] = 0.47 + (payload[0] * 1.5 / 66)       # voltage of energy buffer in Volts
         result['HUM'] = (payload[1] / 250.0 * 100)           # relative humidity in percent
-        result['TMP'] = -20.0 + (payload[2] / 250.0 * 40.0)  # temperature in degree Celsius from -20.0 degC- 60degC
+        result['TMP'] = -20.0 + (payload[2] / 250.0 * 80.0)  # temperature in degree Celsius from -20.0 degC - 60degC
         return result
 
     def _parse_eep_A5_08_01(self, payload, status):
         # Brightness and movement sensor, for example eltako FBH65TFB, RORG = 0x07
+        logger.debug("enocean: parsing A5_08_01: Movement sensor")
```

Comment:
Don't want to create a logger instance in the parser object, too? Global variables are always a litte bit bad practice.